python/tchannel/tornado/connection.py

```diff
@@ -75,7 +76,7 @@ class TornadoConnection(object):
     CALL_REQ_TYPES = frozenset([Types.CALL_REQ, Types.CALL_REQ_CONTINUE])
     CALL_RES_TYPES = frozenset([Types.CALL_RES, Types.CALL_RES_CONTINUE])
 
-    def __init__(self, connection):
+    def __init__(self, connection, tchannel=None):
```

Comment:
When that happens, we can pass that information down. This just seems like a bad violation of concerns.